pyreal/explainers/base.py

```diff
@@ -22,6 +22,16 @@ def _check_transformers(transformers):
     return transformers
 
 
+def _get_transformers(transformers, algorithm=None, model=None, interpret=None):
+    select_transformers = []
+    for t in transformers:
+        if (algorithm is None or t.algorithm == algorithm) \
```

Comment:
I think we should use `True/False` instead of `None` for the logic here since you used `True/False` in `Explainer`

Reply:
For instance, should we still append the transformer when both `t.algorithm` and `algorithm` are `False`?

Reply:
Oh, does `False` stand for inverse transform? If so, I think that should be clarified somewhere.

Reply:
`True` here means return if this flag is True. `False` means return if this place is false. `None` means that this flag value doesn't matter. We need functionality for all these cases later on.

For example, `_get_transformers(model=True, algorithm=True, interpret=None)` will return a transformer that has `(model=True, algorithm=True, interpert=True)` OR `(model=true, algorithm=True, interpret=False)`
`_get_transformers(model=True)` will return any transformer with `(model=True)`, regardless of the other flags